docs/SetupDatabaseForAzure.md

```diff
@@ -1,72 +1,176 @@
+
+* [Backup and restore in Azure Database for PostgreSQL - Flexible Server](https://learn.microsoft.com/en-us/azure/postgresql/flexible-server/concepts-backup-restore)
+
+#### Configure monitoring (Recommended in the production environment)
+
+You can configure the monitoring of Azure Database for PostgreSQL using its native feature. Please refer to the official document for more details.
+
+* [Monitor metrics on Azure Database for PostgreSQL - Flexible Server](https://learn.microsoft.com/en-us/azure/postgresql/flexible-server/concepts-monitoring)
+
+
+#### Disable public access (Recommended in the production environment)
 
-### Steps
+You can configure **Private access (VNet Integration)** as a **Connectivity method**. Please refer to the official document for more details.
 
-You need to choose Flexible or Single server deployment for Azure Database for PostgreSQL.
-* Create an Azure Database for PostgreSQL, 
-  * For single server deployment, follow this [Azure official guide](https://docs.microsoft.com/en-us/azure/postgresql/quickstart-create-server-database-portal).
-  * For flexible server deployment, follow this [Azure official guide](https://docs.microsoft.com/en-us/azure/postgresql/flexible-server/quickstart-create-server-portal).
-* (Optional) Configure advanced monitoring services with the [Azure official guide](https://docs.microsoft.com/en-us/azure/postgresql/concepts-monitoring), monitoring is enabled in Azure Database for PostgreSQL by default.
-* (Optional) To scale resources like vCores and storage after the deployment,
-  * Read the [Azure official guide](https://docs.microsoft.com/en-gb/azure/postgresql/concepts-pricing-tiers#scale-resources) for learning more about scaling single server resources.
-  * Read the [Azure official guide](https://docs.microsoft.com/en-us/azure/postgresql/flexible-server/concepts-compute-storage#scale-resources) for learning more about scaling flexible server resources.
+* [Networking overview for Azure Database for PostgreSQL - Flexible Server](https://learn.microsoft.com/en-us/azure/postgresql/flexible-server/concepts-networking)
 
-Note:-
+You can access the database server from the Scalar product pods on your AKS cluster as follows.
 
-  * The [Storage auto-grow](https://docs.microsoft.com/en-gb/azure/postgresql/concepts-pricing-tiers#storage-auto-grow) feature automatically increases storage without impacting the workload and is enabled by default in server configuration while creating an Azure Database for PostgreSQL instance (not available in flexible server deployment as of now). 
-  
+* Create the database server on the same VNet as your AKS cluster.
+* Connect the VNet for the database server and the VNet for the AKS cluster for the Scalar product deployment using [Virtual network peering](https://learn.microsoft.com/en-us/azure/virtual-network/virtual-network-peering-overview). (// TODO: We need to test this feature with Scalar products.)
```

Comment:
We have not yet tested this feature with Scalar products.
I will do the test and update the document as another task.